blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/CalendarClock.java

```diff
@@ -0,0 +1,21 @@
+package com.rackspacecloud.blueflood.utils;
+
+import org.joda.time.Instant;
+
+import java.util.Calendar;
+
+public class CalendarClock implements Clock {
```

Comment:
Both return the current time in millis which is just a number of millis since epoch. I think we should be fine. 

https://docs.oracle.com/javase/7/docs/api/java/util/Calendar.html#getTimeInMillis()
http://joda-time.sourceforge.net/apidocs/org/joda/time/DateTimeUtils.html#currentTimeMillis()